frontend/src/features/admin-form/responses/common/utils/mrfSubmissionView.ts

```diff
@@ -0,0 +1,31 @@
+import { WorkflowStatus } from '~shared/types'
+
+export enum MRF_STATUS {
+  COMPLETED = 'Completed',
+  PENDING = 'Pending',
+}
+
+/** Gets the business friendly string for current step of workflow. */
+export const getCurrentStepString = (
+  workflowCurrentStepNumber: number | undefined,
+  workflowNumTotalSteps: number | undefined,
+) =>
+  workflowCurrentStepNumber && workflowNumTotalSteps
+    ? `Step ${workflowCurrentStepNumber} of ${workflowNumTotalSteps}`
+    : ''
+
+/** Gets the business friendly string for MRF submission status. */
+export const getStatusFromWorkflowStatus = (
+  workflowStatus: WorkflowStatus | undefined,
+) => {
+  switch (workflowStatus) {
+    case WorkflowStatus.COMPLETED:
+    case WorkflowStatus.APPROVED:
+    case WorkflowStatus.REJECTED:
+      return MRF_STATUS.COMPLETED
+    case WorkflowStatus.PENDING:
+      return MRF_STATUS.PENDING
+    default:
```

Comment:
/nit since it is likely that all the `WorkflowStatus` is fully defined, we would want the [typesystem](https://www.notion.so/opengov/Use-exhaustive-switch-for-enums-e273c275a684492d8aef4789b4882906?pvs=4) to let us know that we need to update the cases here rather than it falling through to default.

Reply:
made it more explicit for `undefined` to become `''`.